### DISCUSSION TITLE
图片拖拽放大问题

### DISCUSSION THREAD

**Discussion by Gerres** [wangeditor-team/wangEditor#4667] [Q&A]
插入图片后可以拖拽放大，但是如果浏览器视图百分比较小情况下，拖拽很容易将图片放大到超过文本域的大小，demo案例中的“模拟腾讯文档”更显而易见（不知道算不算一个小bug），有没有什么办法可以限制图片最大拖拽只能到文本域的width或者height同等大小？

**Comment by wangfupeng1988** [comment]
好建议~
